providers/inwx/inwxProvider.go

```diff
@@ -0,0 +1,292 @@
+package inwx
+
+import (
+	"encoding/json"
+	"fmt"
+	"sort"
+	"strings"
+
+	"github.com/StackExchange/dnscontrol/v3/models"
+	"github.com/StackExchange/dnscontrol/v3/pkg/diff"
+	"github.com/StackExchange/dnscontrol/v3/providers"
+
+	"github.com/svenpeter42/goinwx"
+)
+
+/*
+INWX Registrar and DNS provider
+
+Info required in `creds.json`:
+	- username
+	- password
+	- totp (TOPT code if 2FA is enabled)
```

Comment:
TOPT?  You'll have to edit creds.json every 60 seconds?

Reply:
Unfortunately INWX doesn't support creating scoped API keys like e.g. Cloudflare does. It's only possible to login with the normal account details which also allow actions like deleting or transferring out domains (even via a single API call).
Additionally, only TOTP is supported as two factor authentication.

I don't change my creds.json every 60 seconds but instead use a env variable:
```
	"inwx": {
		"username": "username",
		"password": "password", 
		"totp": "$INWX_TOTP",
	},
``` 
and run `INWX_TOTP=12345 dnscontrol preview`. I'll add this to the documentation as an example later.

I can additionally add `totp-key` where the secret key could be provided and then generate the TOTP based on the current time as an alternative. That's probably more useful when using dnscontrol as part of a CI/CD pipeline.

Reply:
Great way to handle that!!